Dockerfile.local

```diff
@@ -0,0 +1,4 @@
+FROM joplin-base
+
+ENTRYPOINT ["./docker-entrypoint.sh"]
```

Comment:
By keeping this entrypoint in our local Dockerfile, we can still use our LOAD_DATA command without needing to rebuild the image